boa3_test/examples/test_native/methods_copy.py

```diff
@@ -0,0 +1,92 @@
+from typing import Any
```

Comment:
is this file 100% a copy from `boa3_test/examples/test_native/methods.py`?
if it is, why to use a copy?

Reply:
This file is a copy of `methods.py` with the OWNER address being the only difference. I copied `methods.py` because the TestEngine doesn't let me use neo/gas native methods, such as `transfer` and `balanceOf`. To use those methods I needed to create a smart contract that calls neo/gas contracts and since the HTLC needs two different people I used two different smart contracts to represent them, hence, modifying the owner address in the process.